Makefile.toml

```diff
@@ -91,6 +91,8 @@ dependencies = [
     # we have to set up the environment for the emscripten job separately
     # Instead, each of these is called individually.
     "wasm-release",
+    "wasm-test-release",
+    "wasm-compare-worklog-keys",
```

Comment:
Issue: You need to add "wasm-compare-worklog-keys" to CI explicitly; see the comment above.  See https://github.com/unicode-org/icu4x/pull/1460/files#diff-063ca77e012f959eba648db60e6868875fd1e70e5f5ac081193d848f8d61ef32